samples/scaleout/distributor/sample.md

```diff
@@ -67,12 +67,21 @@ The works are simply host for running the handlers defined in `Worker.Handlers`
 snippet:Workerstartup
 
 
-#### Configuration
+## Configuration
+
+### Worker and Distributor on the same machine
 
 snippet: workerConfig
 
 The Node in the MasterNodeConfig points to the host name where the MasterNode is running. If running the Worker from the same machine as the Distributor, Node should equal "localhost".
 
+### Parallel processing of messages
+
+By default NServiceBus (pre V6) processes message sequentially. The following configuration increases the maximum concurrency level to the number of cores of the machine.
```

Comment:
Is the concurrency level == number of cores the recommended approach or a good starting point? If so I think we should say that explicitly.